spec/users_controller_spec.rb

```diff
@@ -0,0 +1,111 @@
+require_relative 'spec_helper'
+
+def app
+  UsersController.new
+end
+
+describe UsersController do
+  before do
+    @users = DB[:users]
+  end
+
+  after do
+    @users.truncate
+  end
+ 
+  #-------------------------------------------------------------------------------------
+  describe "when logged in as an admin" do
+    before do
+      # DB.transaction(:rollback => :always) do
```

Comment:
When I run the tests using DB.transaction, a NoMethodError: undefined method `[]' for nil:NilClass shows up. I'm not sure how to properly implement DB.transaction that's why I used an after block instead.